lib/openai/http.rb

```diff
@@ -53,11 +53,19 @@ def to_json(string)
     # @param user_proc [Proc] The inner proc to call for each JSON object in the chunk.
     # @return [Proc] An outer proc that iterates over a raw stream, converting it to JSON.
     def to_json_stream(user_proc:)
-      proc do |chunk, _|
-        chunk.scan(/(?:data|error): (\{.*\})/i).flatten.each do |data|
-          user_proc.call(JSON.parse(data))
-        rescue JSON::ParserError
-          # Ignore invalid JSON.
+      proc do |chunk, overall_received_bytes, env|
+        Rails.logger.debug("raw chunk: #{chunk}")
```

Comment:
Please remove references to `Rails`.  It's not a dependency and will cause issues for anyone using this outside of Rails.

I've created a new issue (#320) to track adding a logger, and this can be added there.